tests/system/connbeat_collector.py

```diff
@@ -0,0 +1,52 @@
+from BaseHTTPServer import BaseHTTPRequestHandler,HTTPServer
+import simplejson
+
+PORT = 7070
+
+
+class connHandler(BaseHTTPRequestHandler):
+	state = None
+
+	def log_message(self, format, *args):
+		# print("nope")
+		42
```

Comment:
This actually overrides the `log_message` from the superclass, effectively suppressing the request logging